src/containers/About/about.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { ReactComponent as Illustration } from './vector.svg';
+import { ReactComponent as Logo } from './logo.svg';
+import './about.css';
+
+const About = () => {
+  return (
+    <div className="about">
+      <p className="about__heading">About</p>
+      <Illustration />
+      <div>
+        <p className="about__title">Mental Snapp</p>
+        <p className="about__content">
```

Comment:
try to add the static data in static file and imported it here so you don't need to repeat the HTML tags
so you should make a new file and put the statics on it like this 
```js
export default [
  {
    title: 'Mental Snapp',
    descrption:
      'Welcome to Mental Snapp. Because good mental health is an art built on the habit of catching and appreciating ordinary moments. Journalling is a way to do just that - tell your story and guide your thoughts to  a positive and honest frame of mind.',
  },
  {
    title: 'How it works',
    descrption:
      'Our journalling technique is based on a system that has been practiced by monks since the Middle Ages, the Daily Examen. It’s based on three questions: What am I grateful for? Recognising good things. A chance to be thankful. What challenges do I face? Acknowledging - not denying- daily challenges. What am I developing? Noting the character strengths and new skills that can come out of current circumstances.',
  },{
   ....
  }
  .
  .
  .
];

```

Reply:
and in this file import the data and make a loop to it 
like 
```jsx
    {aboutData.map((about) => (
        <div key={ ... }>  {/* put key here */}
          <p className="about__title">{about.title}</p>
          <p className="about__content">{about.descrption}</p>
        </div>
      ))}
```

Reply:
Thanks for explanation , I will do it